File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height, row, column, spaces;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (row = 0; row < height; row += 1)
    {
        for (spaces = 0; spaces < height - row - 1; spaces += 1)
        {
            printf(" ");
        }
        for (column = 0; column <= row; column += 1)
        {
            printf("#");
        }
        printf("  ");
        for (column = 0; column <= row; column += 1)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break

for row in range(height):
    for spaces in range(height - (row+1)):
        print(" ", end='')
    for col in range(row+1):
        print("#", end='')
    print("  ", end="")
    for col in range(row+1):
        print("#", end='')
    print()